test/e2e/generated_clientset.go

```diff
@@ -187,6 +188,124 @@ var _ = framework.KubeDescribe("Generated release_1_5 clientset", func() {
 	})
 })
 
+func observeJobCreation(w watch.Interface) {
```

Comment:
OT: instead of having multiple observe* why not having single generic function doing this, especially you're using `watch.Interface` as an argument.